src/components/SingleProduct/SingleProducts.css

```diff
@@ -0,0 +1,57 @@
+@import "../../shared.css";
+@import "../Cart/Cart.css";
+/* Sections */
+.section-product {
+  width: 100%;
+  height: 75vh;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  gap: 20px;
+  align-items: flex-start;
+  color: var(--primary-color);
+}
+.right-section {
+  background-color: var(--txt-color);
+}
+
+.left-section-products {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: fit-content;
+  background-color: var(--txt-color);
+}
+
+/* Image dimension */
+.img-height {
+  height: 350px;
+}
+
+/* Product details */
+.product-details {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+.strikethrough {
+  text-decoration: line-through;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  border-bottom: 1px solid var(--primary-color);
+}
```

Comment:
You can make a utility CSS class for properties that are getting repeated often, for instance: `display: flex;`

Reply:
ok will make the changes